.github/_support/github-slack-notify.py

```diff
@@ -118,11 +118,15 @@ def build_payload(status):
 
 def on_main_repo():
     """check if running from a fork"""
-    return os.environ["GITHUB_REPOSITORY"] == "funcx-faas/funcx"
+    res = os.environ["GITHUB_REPOSITORY"].lower() == "funcx-faas/funcx"
+    print(f"Checking main repo: {res}")
+    return res
 
 
 def should_notify(status):
-    return check_status_changed(status) and on_main_repo() and not is_pr_build()
+    res = check_status_changed(status) and on_main_repo() and not is_pr_build()
+    print(f"Should notify: {res}")
```

Comment:
Are these print statements here to help with debugging?

Reply:
Yeah, the action prints each of these to explain the process, except there weren't any in the decision functions. These two prints just add insight into what those two decisions are.